_trumps.spacing-responsive.scss

```diff
@@ -1,3 +1,4 @@
+@charset "utf-8";
```

Comment:
I tried that back then. Seems to have been resolved now though. Not sure if it's due to version updates.